config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -42,7 +43,7 @@ presubmits:
         - --aksengine-public-key=$(K8S_SSH_PUBLIC_KEY_PATH)
         - --aksengine-private-key=$(K8S_SSH_PRIVATE_KEY_PATH)
         - --aksengine-winZipBuildScript=$(WIN_BUILD)
-        - --aksengine-orchestratorRelease=1.24
+        - --aksengine-orchestratorRelease=1.23
```

Comment:
I suppose this should be kept as 1.24

Reply:
yes, it is required to be on 1.24 for some feature that were depreciated.